components/web/goaccess/manifests/sample-manifest.p5m

```diff
@@ -22,14 +22,15 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
-file path=etc/goaccess.conf
+file path=etc/goaccess/browsers.list
+file path=etc/goaccess/goaccess.conf
```

Comment:
Please include preserve=true to config files the user should be able to update, otherwise a "pkg fix" call to restore system state will overwrite user changes.

Reply:
Thanks Toasterson, now it's included preserve=true in config files